iocore/net/P_OCSPStapling.h

```diff
@@ -21,13 +21,14 @@
 
 #pragma once
 
+#include "ts/ink_config.h"
+
+#if TS_USE_TLS_OCSP
 #include <openssl/ssl.h>
+#include <openssl/ocsp.h>
 
-#ifdef OCSP_sendreq_new
 #define HAVE_OPENSSL_OCSP_STAPLING 1
```

Comment:
Should we keep HAVE_OPENSSL_OCSP_STAPLING? I think we can replace all HAVE_OPENSSL_OCSP_STAPLINGs with TS_USE_TLS_OCSP.

Reply:
Good idea! Updated patch.